src/main/java/pageObjects/MenuPage.java

```diff
@@ -1,53 +1,36 @@
 package pageObjects;
 
-import browser.Driver;
-import org.junit.Assert;
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.FindBy;
-import pageObjects.AbstractPage;
 import lombok.Getter;
-import lombok.Setter;
 import org.openqa.selenium.WebElement;
-import utlis.RandomListSelect;
+import org.openqa.selenium.support.FindBy;
 
 import java.util.List;
 
 
 @Getter
-@Setter
-
 public class MenuPage extends AbstractPage {
     public MenuPage() {
         super();
     }
 
     @FindBy(xpath = "//div[@class='modal-menu modal-menu_open']//a[*]")
-    private  List<WebElement> menuList;
+    private List<WebElement> menuList;
 
+    @FindBy(xpath = "//div[@class='product__img show_order_win']")
+    private List<WebElement> productPhotoList;
 
     @FindBy(xpath = "//div[@class='product__buy add_to_cart']")
-    private  List<WebElement> productList;
+    private List<WebElement> productList;
 
     @FindBy(xpath = "//div[@class='product__alert_msg']")
-    private  WebElement productAlertMessage;
-
-
-    public  List<WebElement> getMenuList() {
-        return menuList;
-    }
-
-    public  List<WebElement> getProductList() {
-        return productList;
-    }
-
+    private WebElement productAlertMessage;
 
     private WebElement lunchButton1;
 
 
     @Override
     public String getUrl() {
-        url = "sefse";
+        url = "https://www.andys.md/en";
```

Comment:
create field for every page instead of overriding getUrl() with the same page in every page object class